src/tests/system/tests/test_authentication.py

```diff
@@ -15,103 +15,115 @@
 
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
 @pytest.mark.parametrize("method", ["su", "ssh"])
-def test_authentication__using_a_good_then_bad_password(client: Client, provider: GenericProvider, method: str):
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
```

Comment:
This is sssd-2.10 specific, please don't pull this into sssd-2.9 branch

Reply:
Removed.